### PR TITLE
Rsanchez/mp update

### PR DESCRIPTION
**Pull request by kaosctrl**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "distorm"]
+	path = distorm
+	url = https://github.com/gdabah/distorm.git
```
Comment (v-wiv) [inline .gitmodules]: Are we cleared to use whatever bits are being pulled in from this `distorm` submodule?

Comment (pwiest) [reply .gitmodules]: Do we know what feature this disassembler is needed for? A bit worry-some that MP is now fiddling with assembly...

Comment (v-wiv) [reply .gitmodules]: According to the updated readme.md file, it appears to be used when "Dynamic Instrumentation" is enabled. https://github.com/Mojang/microprofile/tree/rsanchez/MP_update#dynamic-instrumentation

Looks like that functionality is disabled by default?

Comment (kaosctrl) [reply .gitmodules]: I believe that is correct.  I was going to look at the new features but wanted to get more eyes on it also. :-)

**Pull request merged**